packages/web/src/app/layout.tsx

```diff
@@ -37,7 +40,34 @@ export async function generateMetadata(): Promise<Metadata> {
 
   // Force dynamic metadata generation by adding timestamp
   const timestamp = Date.now();
-  const config = getDaoConfigServer();
+
+  let config: Config | null = null;
+  if (isRemoteApiConfigured()) {
+    try {
+      const host = await getRequestOrigin();
+      if (!host) {
+        throw new Error("No host found");
+      }
+      const response = await fetch(`${host}/api/config`);
```

Comment:
you should put host to fetch header `x-degov-site`, the request api is `${process.env.NEXT_PUBLIC_DEGOV_API}/dao/config`